src/test/java/lessonSixCalculator/CalculatorTest.java

```diff
@@ -0,0 +1,31 @@
+package lessonSixCalculator;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class CalculatorTest {
+
+    @Test
+    public void shouldAddTwoNumbers(){
```

Comment:
shouldAddTwoNumbers -> shouldAdd

Nie musimy dodawać, że dodajemy dwie liczby. A wręcz umieszczenie tego w nazwie może być mylące, ponieważ sugeruje, że jest to jakiś wyjątkowy przypadek zastosowania naszej metody.

Reply:
Analogicznie pozostałe testy.

Reply:
zrobione